testing/gearmand/gearmand.post-deinstall

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+## print notice
```

Comment:
You're not the only one.  Buddy of mine in the office keeps threatening to write a de-commenter for me.